src/app/pages/charts/charts.module.ts

```diff
@@ -4,16 +4,14 @@ import { FormsModule } from '@angular/forms';
 
 import { ThemeModule } from 'theme/theme.module';
 
-import {
-  ChartsComponent,
-  BrowserStatisticsChartModule,
-  CountryStatisticsChartModule,
-  DiscreteBarChartModule,
-  LineChart1Module,
-  LineChart2Module,
-  LinePlusBarChartModule,
-  StackedBarChartModule,
-} from '.';
+import { ChartsComponent } from './charts.component';
```

Comment:
why did you decide to separate import?

Reply:
Because there is no specific rule with which I could decide whether it is necessary to collect imports into index.ts. It's inconsistent. I don't see the point in this, it's just extra lines for me.